src/js/@llrt/runtime.ts

```diff
@@ -37,7 +37,7 @@ if (!AWS_LAMBDA_RUNTIME_API) {
   );
 }
 
-const HANDLER_ENV = _HANDLER || LAMBDA_HANDLER;
+const HANDLER_ENV = LAMBDA_HANDLER || _HANDLER;
```

Comment:
Had to make this change for `sam local start-api` to work since it adds `aws-lambda-rie` to the image and changes `ENTRYPOINT` if not specified. The entry point will then pass the `CMD` from the docker file as first argument to the executable and set it as `_HANDLER`. `_HANDLER` is reserved in Lambda even for container images so have to use `LAMBDA_HANDLER` instead